Automattic/jetpack
# [Donation Block] Default to the Stripe Currency on Mount and Display a  ✔️ next to the selected currency

Fixes https://github.com/Automattic/gold/issues/39

## Proposed changes:

*  Display a  ✔️ next to the selected currency
* Default to the Stripe currency on mount. 

### Other information:


## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Sync changes to WPCOM
* Change the Stripe account Currency connected to the site to any currency besides USD or you can change the default value in `donations.php` for the `currency` attribute to another currency
* Add the Donation block to a post
* Notice the Stripe default currency is selected. In most cases, this will be USD
* Select a currency and notice that  the ✔️ icon is to the right of the currency
* Save the post and notice no errors are thrown
* Change the currency to another currency and save the post and notice no errors are thrown
* Refresh the page and notice that the user-selected currency isn't overridden by the Stripe currency
